### ISSUE TITLE
Muted Findings refactoring

### ISSUE THREAD

**Issue by Rieven** [minvws/nl-kat-coordination#680]
**Notes:**
* UX is conceptually dubious: Mute objects shows up in Rocky's interface although they are not semantically similar to other objects. They currently have scan levels for example, which doesn't really make sense. This should be refactored/reworked later. 
* Currently no way to specify how long an object should be muted (requirement fron Z-cert). We should probably make a new ticket+PR for that. Gotta keep PR's small :)

_Originally posted by @Darwinkel in https://github.com/minvws/nl-kat-coordination/issues/674#issuecomment-1498628161_

**Comment by underdarknl** [comment]
>  UX is conceptually dubious: Mute objects shows up in Rocky's interface although they are not semantically similar to other objects. They currently have scan levels for example, which doesn't really make sense. This should be refactored/reworked later.

I agree, they should be hidden from the usual lists, they should not have (visible) scan levels, and they should generaly present themselves as a property of the hidden finding instead of displaying an extra object page.

>  Currently no way to specify how long an object should be muted (requirement fron Z-cert). We should probably make a new ticket+PR for that. Gotta keep PR's small :)

This functionality needs to be added to the Octopoes api, so we can add objects (any) with a valid_till timestamp. Once this is done we can add the timestamp selector (or handy buttons like 6 months from now) to this form.

**Comment by zcrt** [comment]
For the second bullet (specifying how long a finding should be muted); it is desirable that when the muted finding gets to be unmuted that that is shown with the finding (so that the operator knows this was a previously muted finding for reason _x_)
